aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -267,7 +267,14 @@ Tensor qadd_scalar_tensor_out(Tensor qa, Tensor b, Tensor out) {
 
 TORCH_LIBRARY_IMPL(quantized, QuantizedCPU, m) {
   m.impl("add",                 TORCH_FN(qadd</*ReLUFused=*/false>));
+  m.impl("add.out",             TORCH_FN(qadd_out</*ReLUFused=*/false>));
+  m.impl("add.Scalar",          TORCH_FN(qadd_scalar</*ReLUFused=*/false>));
```

Comment:
Why `Scalar` and not `scalar`?

Reply:
this is following the naming of other overloaded functions in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml